server/server.py

```diff
@@ -318,6 +328,29 @@ async def details(request: Request):
 
 
 def worker(sentence, speaker_id, model, use_aliases, new_speaker_ids):
+    def substitute_time(sentence):
+        # Regular expression to find time pattern (HH:MM)
+        time_pattern = re.compile(r'(\d{1,2}):(\d{2})')
```

Comment:
throws value errors in the case 154:00 or 15:001, correct regex should be `(\s\d{1,2}):(\d{2}\s)`